logging: Fix musl build issue with fcntl

 error: #warning redirecting incorrect #include <sys/fcntl.h> to <fcntl.h> [-Werror=cpp]
 #warning redirecting incorrect #include <sys/fcntl.h> to <fcntl.

Upstream-Status: Pending
Signed-off-by: Armin Kuster <akuster@mvista.com>

Index: git/src/swtpm/logging.c
===================================================================
--- git.orig/src/swtpm/logging.c
+++ git/src/swtpm/logging.c
@@ -43,7 +43,7 @@
 #include <errno.h>
 #include <string.h>
 #include <sys/types.h>
-#include <sys/fcntl.h>
+#include <fcntl.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
